.github/workflows/code_quality.yml

```diff
@@ -1,22 +1,35 @@
-name: code_quality
+name: Code quality validation
 
 on:
   push:
   pull_request:
 
 
 jobs:
-  build:
+  code_validation:
+    name: Code quality validation
     runs-on: ubuntu-latest
+
     steps:
       - uses: actions/checkout@v2
+
       - name: Install Python 3
         uses: actions/setup-python@v1
         with:
           python-version: 3.8
+
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements.txt
+
+      - name: Set up Cloud SDK for access to google sheets
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          version: '290.0.1'
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          service_account_email: ${{ secrets.GCP_SA_EMAIL }}
+          export_default_credentials: true
+
       - name: Run tests with pytest
         run: pytest
```

Comment:
this runs the test suite in etl-pipeline/tests/ correct?

Reply:
yes as defined in the pytest.ini (else it would try to discover all tests in all subfolders of the root, and in our case we don't want to run the full intergration tests every time as not to burdon 3th parties like covid19india.org api)
added a small update to the readme.md for pytest.ini to reflect that